docs/concepts/flows.md

```diff
@@ -1103,66 +743,58 @@ async def greet_user():
     print(f"Hello, {user_input.name}!")
 ```
 
-Running this flow will create a flow run. The flow run will advance until code execution reaches `pause_flow_run`, at which point it will move into a `Paused` state.
-Execution will block and wait for resumption.
+Running this flow creates a flow run. The flow run advances until code execution reaches `pause_flow_run`, at which point it moves into a `Paused` state.
+Execution blocks and waits for resumption.
 
-When resuming the flow run, users will be prompted to provide a value for the `name` field of the `UserNameInput` model.
-Upon successful validation, the flow run will resume, and the return value of the `pause_flow_run` will be an instance of the `UserNameInput` model containing the provided data.
+When resuming the flow run, users must provide a value for the `name` field of the `UserNameInput` model.
+Upon successful validation, the flow run resumes, and the return value of the `pause_flow_run` is an instance of the `UserNameInput` model containing the provided data.
 
-For more in-depth information on receiving input from users when pausing and suspending flow runs, see the [Creating interactive workflows](/guides/creating-interactive-workflows/) guide.
+For more information on receiving input from users when pausing and suspending flow runs, see [Creating interactive workflows](/guides/creating-interactive-workflows/).
 
-## Canceling a flow run
+### Cancel flow runs
 
-You may cancel a scheduled or in-progress flow run from the CLI, UI, REST API, or Python client.
+Cancel a scheduled or in-progress flow run from the CLI, UI, REST API, or Python client to transition flow runs to a `Cancelling` state.
 
-When cancellation is requested, the flow run is moved to a "Cancelling" state.
-If the deployment is a work pool-based deployemnt with a worker, then the worker monitors the state of flow runs and detects that cancellation has been requested.
-The worker then sends a signal to the flow run infrastructure, requesting termination of the run.
-If the run does not terminate after a grace period (default of 30 seconds), the infrastructure will be killed, ensuring the flow run exits.
+How it works:
```

Comment:
I would leave this section as is. I think the proposed changes don't capture the useful information. 

One change that would be good is saying that a deployment is required up front.